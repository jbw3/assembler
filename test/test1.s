MOVI 1
MVR0 R1
MOVI 2
MVR0 R2

MOVI 4
MVR0 R3
MOVI 4
SHLL R2
ADD  R3
MVR0 R6

MOVI 1
MVR0 R3
MOVI 4
SHLL R2
ADD  R3
MVR0 R7

MOVI 3

BEZ  R6
SUB  R1
JMP  R7

NOP
