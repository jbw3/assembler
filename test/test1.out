11
19
12
1a
14
1b
14
52
43
1e
11
1b
14
52
43
1f
13
76
49
6f
00
